php_phongo.h

```diff
@@ -112,35 +112,43 @@ void phongo_throw_exception(php_phongo_error_domain_t domain TSRMLS_DC, const ch
 ;
 void phongo_throw_exception_from_bson_error_t(bson_error_t *error TSRMLS_DC);
 
-/* This enum is used for libmongoc function selection for the
- * phongo_execute_command types. The values are important, as the READ and
- * WRITE fields are also used as a bit field to see whether ReadPreference,
- * ReadConcern, and WriteConcern are supported for each type. */
+/* This enum is used for processing options in phongo_execute_parse_options and
+ * selecting a libmongoc function to use in phongo_execute_command. The values
+ * are important, as READ and WRITE are also used as a bit field to determine
+ * whether readPreference, readConcern, and writeConcern options are parsed. */
 typedef enum {
-	PHONGO_COMMAND_RAW =        0x13,
-	PHONGO_COMMAND_READ =       0x01,
-	PHONGO_COMMAND_WRITE =      0x02,
-	PHONGO_COMMAND_READ_WRITE = 0x03
+	PHONGO_OPTION_READ_CONCERN    = 0x01,
+	PHONGO_OPTION_READ_PREFERENCE = 0x02,
+	PHONGO_OPTION_WRITE_CONCERN   = 0x04,
+	PHONGO_COMMAND_RAW            = 0x07,
+	PHONGO_COMMAND_READ           = 0x03,
+	PHONGO_COMMAND_WRITE          = 0x04,
```

Comment:
Are you sure you can have duplicate enum values?

Reply:
Yessir. See [this Stack Overflow thread](https://stackoverflow.com/questions/5561142/is-it-legal-to-have-duplicate-enum-values-in-c).